Ali_Nurisiahsiahi.md

```diff
@@ -0,0 +1,7 @@
+I am Ali Nurisiahsiahi
+I like cycling,bodybuilding and walking because help my helth.
+My fouriy food are Spageti and Ghorma zabzi(Iranian food) becuse are delicious.
+My favorit song is Afrid and singer is Eminem https://www.youtube.com/watch?v=j5-yKhDd64s
```

Comment:
Try to use [markdown links](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links), not just plain URLs